src/main/java/com/github/splendor_mobile_game/websocket/utils/Log.java

```diff
@@ -23,6 +34,25 @@ public static void setSavingLogsToFile(String logsDir) {
         Log.fileToWrite = new File(path.toString());
         Log.fileToWrite.getParentFile().mkdirs();
         Log.savingToFile = true;
+
+        fileLogLevels=EnumSet.allOf(LogLevel.class);
+        consoleLogLevels=EnumSet.allOf(LogLevel.class);
```

Comment:
You have placed these lines in the `setSavingLogsToFile` method. And it's not guaranteed that it will be called before using the Logger. User may want not to use this method.

Default settings should be that everything is printed to the console, even without calling this method.